buildroot/x86_64.config

```diff
@@ -6,4 +6,7 @@ BR2_TARGET_GENERIC_ROOT_PASSWD="root"
 BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="../overlay-poweroff"
 BR2_TARGET_ROOTFS_CPIO=y
+BR2_TARGET_ROOTFS_EXT2=y
```

Comment:
so this will build 2 images now by having this and the _CPIO config above? (That's fine, just curious if that's how it works)

Reply:
Correct!